ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -1832,7 +1832,14 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
           if (window.confirm("$services.localization.render('fileManager.action.deleteFilesConfirmation')")) {
             // Clear the selection.
             scope.selection = {};
-            scope.drive['delete'](getPaths(files)).done(refreshLiveTable);
+            scope.drive['delete'](getPaths(files)).done(function(result) {
+              if (result.error) {
+                  var message = $jsontool.serialize( ##
```

Comment:
This is not valid JavaScript code. When using Velocity code inside JavaScript code you need to make sure the JavaScript code remains valid. For instance, if this code would have been in a JavaScript file then this file could not be minified because the JavaScript is not valid.

In this case the solution for the max line length is quite simple: wrap the line as if it were a JavaScript line. For instance:

```
var message = $jsontool.serialize($services.localization.render('fileManager.action.deleteError',
  ['__error__']));
```

or

```
var message = $jsontool.serialize($services.localization.render(
  'fileManager.action.deleteError', ['__error__']));
```

or simply remove the ``##`` :)

Reply:
Thanks I have done the last suggestion.

I had tested that it produced valid javascript and it did. 
Are you telling me that the string content of the JSX' code property also needs to be valid javascript as it could be that some JS-oriented tool such as UglifyJS operates on that content before velocity is run?
thanks
Paul

Reply:
The Velocity code is evaluated before the minification of the JavaScript code in case of a JSX, which allows you to embed Velocity code without caring if it looks as JavaScript code, but that doesn't mean it's a good practice. The code won't work if you move it to a resource file later. The best practice is to make sure the JavaScript code is valid even before the execution of the Velocity code (e.g. by putting the Velocity code in a JavaScript comment).

Reply:
Thanks.